cmd/gaia/app/genesis.go

```diff
@@ -32,6 +31,7 @@ var (
 // State to Unmarshal
 type GenesisState struct {
 	Accounts  []GenesisAccount   `json:"accounts"`
+	Txs       []json.RawMessage  `json:"txs"`
```

Comment:
[]StdTx?

Reply:
Yeah that was the original implementation. I could roll it back, though this is not blocking now